pkg/importer/util_test.go

```diff
@@ -32,23 +67,49 @@ func TestParseEndpoint(t *testing.T) {
 	type args struct {
 		endpt string
 	}
+
 	tests := []struct {
 		name    string
 		args    args
-		want    *url.URL
+		want    string
+		setEnv  bool
 		wantErr bool
 	}{
-		// TODO: Add test cases.
+		{
+			name:    "successfully get url object from endpoint",
+			args:    args{"http://www.bing.com"},
+			want:    "good value",
+			setEnv:  true,
+			wantErr: false,
+		},
+		{
+			name:    "successfully get url object from default value",
+			args:    args{""},
+			want:    "good value",
+			setEnv:  true,
+			wantErr: false,
+		},
+		{
+			name:    "return error with empty endpoint and no default endpoint set",
+			args:    args{""},
+			want:    "",
```

Comment:
`want` should be a bool value, currently it's used to signify `len(want) > 0` for wanting any non-zero length string